notebooks/anomaly_detection/LODA_anomaly_detection.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<ul><li>&nbsp;"Loda constructs&nbsp;k&nbsp;one-dimensional histograms <strong>hii=1" </strong>bold<strong> </strong>part might need curly brackets between the two i's.</li><li>"with larg scores (nnl)." it might be an idea to write this part explicitly. I couldn't figure out the "nnl" part. Please ignore if it's something obvious.</li><li> Is this P sub i and P sub i ?&nbsp;"a fixed-width histogram density estimator pipi is estimated heuristically or set from hyperparameter" Assuming problem is not in the browser</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed and updated the issues.